includes/class-wc-emails.php

```diff
@@ -611,6 +611,16 @@ public function low_stock( $product ) {
 			return;
 		}
 
+		/**
+		 * Action hook fired to determine if the current product should trigger a
+		 * low stock notification
+		 *
+		 * @param WC_Product $product - The out of stock product object
+		 *
+		 * @since 4.4.0
+		 */
+		do_action( 'woocommerce_should_notify_low_stock', $product );
```

Comment:
This hook name doesn't sounds correct, you can't control using. Also it sounds very specif for only your user case. It should be sounds better like:

```suggestion
		do_action( 'woocommerce_before_low_stock_notification', $product );
```

Reply:
@JohnTendik Now that it is a filter `woocommerce_should_send_low_stock_notification`.